docs/advanced-topics/wes/wes-agc-tutorial.rst

```diff
@@ -0,0 +1,332 @@
+WES Tutorial for AWS AGC
+========================
+Amazon Web Services' (AWS) Amazon Genomics CLI (AGC) is a command line tool for launching cloud infrastructure
+within AWS accounts that can be used to execute genomics workflows. The infrastructure deployed by AGC implements the WES
+standard, and thus can be directly communicated with by using the Dockstore CLI.
+
+Check out the official AGC `GitHub page <https://github.com/aws/amazon-genomics-cli>`_.
+
+Download and Install AWS AGC
+----------------------------
+AGC provides a `quick-start guide <https://aws.github.io/amazon-genomics-cli/docs/getting-started/>`_ for initial setup
+and getting familiar with the tool. The following workflow execution tutorials will cover all steps for using AGC once
+it has been `installed <https://aws.github.io/amazon-genomics-cli/docs/getting-started/installation/>`_.
+
+Tutorial Topics
+----------------
+The following WDL workflow tutorials will cover:
+
+1. Deploying an AGC project and context
+2. Configuring the Dockstore CLI to communicate with AGC infrastructure
+3. Launching a workflow
+
+.. note::
+
+    1. AWS AGC requires a special JSON file to be passed as the test parameter file for workflows, this JSON is formatted as follows:
+
+        .. code:: text
+
+            {
+                "workflowInputs": <pathToInputJSON>
+            }
+
+    2. AWS AGC references descriptor files by URLs. The Dockstore CLI references the primary descriptor in an execution request as a GA4GH Tool Registry Service (`TRS <https://github.com/ga4gh/tool-registry-service-schemas>`_) URL.
+    The AGC infrastructure will only be able to access the file referenced by the TRS URL if the workflow is publicly published on Dockstore.
```

Comment:
`publicly published` is kinda redundant. 
Maybe `published (i.e. visible to the public)` or just `published (i.e. public)`

Reply:
Fixed: https://github.com/dockstore/dockstore-documentation/pull/173/commits/5fc7d40dcd443fe5cccb7eab086c39f6cdb9a78f